evalml/automl/automl_search.py

```diff
@@ -188,10 +189,14 @@ def __init__(
             tolerance (float): Minimum percentage difference to qualify as score improvement for early stopping.
                 Only applicable if patience is not None. Defaults to None.
 
-            allowed_pipelines (list(class)): A list of PipelineBase subclasses indicating the pipelines allowed in the search.
-                The default of None indicates all pipelines for this problem type are allowed. Setting this field will cause
+            allowed_component_graphs (list): A list of dictionaries or ComponentGraphs indicating the component graphs allowed in the search.
+                The format should follow [ {Name_of_graph: [list_of_components],
+                                            random_seed: 42} ].
+                The default of None indicates all pipeline component graphs for this problem type are allowed. Setting this field will cause
                 allowed_model_families to be ignored.
 
+                e.g. allowed_component_graphs = [ {"Name": ["Imputer", "One Hot Encoder", dummy_classifier_estimator_class]} ]
```

Comment:
Nit-pick: I don't think we should use `dummy_classifier_estimator_class` since it's a test fixture.